Documentation/Blazorise.Docs/Pages/Blog/UsingTheSelectComponent/Examples/UsingTheSelectComponentComplexTypeExample.razor

```diff
@@ -0,0 +1,42 @@
+﻿@namespace Blazorise.Docs.Pages.Blog.UsingTheSelectComponent.Examples
+
+<Row>
+    <Column>
+        <Select TValue="int"
```

Comment:
When you have any input component tend to wrap it with a `Field`. That is always recommended.